Libraries/MiscDrivers/Display/tft_st7735.h

```diff
@@ -216,12 +216,25 @@ void MXC_TFT_ResetCursor(void);
  *
  * @param       x0              x location on screen
  * @param       y0              y location on screen
- * @param       fon_id          Font number
+ * @param       id              Font number ID
```

Comment:
Should be `@param font_id`, or rename the variable in the function declaration to `id`.